dist/src/main/java/io/camunda/application/commons/configuration/GatewayBasedConfiguration.java

```diff
@@ -87,11 +86,6 @@ public BrokerClientTimeoutConfiguration brokerClientConfig() {
     return new BrokerClientTimeoutConfiguration(properties.getCluster().getRequestTimeout());
   }
 
-  @Bean
-  public MultiTenancyCfg multiTenancyCfg() {
```

Comment:
Can we remove this class or is still used somewhere?

Reply:
yes I was planning to remove it, then I forgot :/

Reply:
I can answer myself: it's still used for backwards compatibility.

Reply:
No it is not need, we can remove it
the backward compatibility is ensured by
https://github.com/camunda/camunda/blob/f8a4f495cf456d140a6983b197a5e2baf046e484/dist/src/main/resources/application-broker.properties#L22
and
https://github.com/camunda/camunda/blob/f8a4f495cf456d140a6983b197a5e2baf046e484/dist/src/main/resources/application-gateway.properties#L17

Reply:
There are many references to various *_MULTITENANCY_ENABLED env vars. Will these continue to work or do we need to update them all? For example, there are references to ZEEBE_BROKER_GATEWAY_MULTITENANCY_ENABLED, ZEEBE_GATEWAY_MULTITENANCY_ENABLED and CAMUNDA_OPERATE_MULTITENANCY_ENABLED.

Reply:
no need to update any parameters used by the standalone applications: if you set 
- `ZEEBE_BROKER_GATEWAY_MULTITENANCY_ENABLED` in a standalone broker, `CAMUNDA_SECRITY_MULTITENANCY_ENABLED` will be set
- `ZEEBE_GATEWAY_MULTITENANCY_ENABLED` in a standalone gateway, `CAMUNDA_SECRITY_MULTITENANCY_ENABLED` will be set
- `CAMUNDA_OPERATE_MULTITENANCY_ENABLED` in a standalone gateway, `CAMUNDA_SECRITY_MULTITENANCY_ENABLED` will be set